docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+services:
+  postgres:
+    image: postgres:latest
+    container_name: postgres
+    environment:
+      POSTGRES_DB: rospi
+      POSTGRES_USER: rospi
+      POSTGRES_PASSWORD: rpitele
+    ports:
+      - "5432:5432"
+    volumes:
+      - /Users/devashishraj/Desktop/postgresGO/raspi_IOT/rpi_telemetery/schema.sql:/docker-entrypoint-initdb.d/schema.sql
```

Comment:
use `$HOME` or relative path here. others systems are not going to have the same dir structure.